ios/RNLinksdk.m

```diff
@@ -48,6 +50,62 @@ @interface RNLinksdk ()
 
 #pragma mark -
 
+// Category to ensure both the old, typo spelling of `insitution` and
+// the corrected spelling `institution` are visible to the implementation
+// to allow compiling against any LinkKit dependency
+@interface PLKSuccessMetadata (InstitutionTypoFix)
+
+- (PLKInstitution * __nonnull)institution;
+- (PLKInstitution * __nonnull)insitution;
+
+@end
+
+// Class to have a distinct +load method to hook into the runtime before
+// SDK logic executes
+@interface PLKSuccessMetadataTypoFix : NSObject
+
+@end
+
+@implementation PLKSuccessMetadataTypoFix
+
++ (void)load {
+    static dispatch_once_t onceToken;
+    // dispatch_once out of an abundance of caution in case +load is ever called multiple times
+    dispatch_once(&onceToken, ^{
+        Class targetClass = NSClassFromString(@"PLKSuccessMetadata");
```

Comment:
I think it would be benefical to assert `targetClass != Nil` (yes [`Nil`](https://nshipster.com/nil/)) and exit early if the precondition isn't met.